### PR TITLE
Resolve Submit Button Bugs and Deploy new MozDef Rule Logic

### PR DESCRIPTION
**Pull request by tristanweir**

### CODE REVIEW

File: rules/duosecurity.js [hunk 1]
```diff
@@ -16,6 +16,13 @@ function (user, context, callback) {
       ignoreCookie: false,
       username: user.email,
     };
+  } else {
```
Comment (gdestuynder) [inline rules/duosecurity.js]: AFAIK this is now fixed by Auth0 and the work-around is no longer needed. @gene1wood ?

Comment (gene1wood) [inline rules/duosecurity.js]: In looking at the [Auth0 support ticket](https://support.auth0.com/tickets/17769) it's not clear what the fix was or if it was deloyed in our instances. I've asked for clarification.

**Comment by gene1wood** [comment]
What does the new MozDef rule logic do? Which submit button bugs does this PR resolve? Can you rebase this so it can be merged?

**Comment by gdestuynder** [comment]
@tristanweir ^^ gene's comment

**Comment by gene1wood** [comment]
PR #80 which is included in this PR was not reviewed

**Pull request closed without merging**